spec/pluralrules.html

```diff
@@ -27,6 +27,8 @@
         1. Perform ? SetNumberFormatOptions(_pluralRules_, _options_, *0*, *3*).
         1. Let _r_ be ResolveLocale(*%PluralRules%*.[[AvailableLocales]], _requestedLocales_, _opt_).
         1. Set _pluralRules_.[[Locale]] to the value of _r_.[[Locale]].
+        1. Let _pluralCategories_ be a List of Strings representing the possible results of plural selection.
+        1. Set _pluralRules_.[[PluralCategories]] to CreateArrayFromList(_pluralCategories_).
```

Comment:
Yes, that would be a way to specify cloning each time. But in the related thread with @erights, I thought we came ot the conclusion that we don't need to do that cloning.